src/base/types/FluxVariabilitySummary.jl

```diff
@@ -0,0 +1,70 @@
+"""
+    FluxVariabilitySummary
+
+A struct used to store summary information about the 
+solution of a flux variability analysis result.
+"""
+struct FluxVariabilitySummary
+    biomass_fluxes :: Dict{String, Vector{Union{Float64, Nothing}}}
+    exchange_fluxes :: Dict{String, Vector{Union{Float64, Nothing}}} 
+end
+
+"""
+    flux_variability_summary(flux_result::Tuple{Dict{String, Dict{String, Float64}}, Dict{String, Dict{String, Float64}}}; 
+        exclude_exchanges = false,
+        exchange_prefixes = _constants.exchange_prefixes,
+        biomass_strings = _constants.biomass_strings,
+        exclude_biomass = false,
+        )
+Return a `FluxVariabilitySummary` struct based on the `flux_result` of a
```

Comment:
Summarize a dictionary of flux dictionaries obtained eg. from flux_variability_analysis_dict. The simplified summary representation is useful for pretty-printing and easily showing the most important results.